src/Components/SignIn.tsx

```diff
@@ -9,9 +9,19 @@ const SignIn = () => {
 
   const navigate = useNavigate()
 
-  const { events, setOrg, setEvents, org } = useContext(EventContext)
-  console.log('signin: ', events)
+  const { events, setOrg, setEvents, org, allOrgs, setAllOrgs } = useContext(EventContext)
+  // console.log('signin: ', events)
 
+  useEffect(() => {
+    apiCalls.loadAllOrganizations()
+      .then(data => setAllOrgs(data.data))
+  })
+
+  const orgNames = allOrgs.map((org: { name: string, id: number }) => {
+    return (
+      <option key={org.id} value={org.id}>{org.name}</option>
```

Comment:
would we want to add any data cy labels to these option elements for cypress testing? perhaps data-cy='{org.name}' or something?